fr.ffs2play.ffs2play.yml

```diff
@@ -0,0 +1,73 @@
+app-id: fr.ffs2play.ffs2play
```

Comment:
Given it's open source and written in packaging-friendly language, is there any reason why it's not build from source?

Reply:
im try to build from source but im not expert of flatpak

Reply:
> @liberodark are upstream involved in distributing this? If not, we need to ask them! If you could put a link here to an issue or conversation doing that it'd be awesome.
> 
> Similarly, we need to send the appdata file upstream, it's generally useful for all Linux distribution types, not just flatpaks and we don't want to host 'improvements' or changes here where we can help it.



> @liberodark are upstream involved in distributing this? If not, we need to ask them! If you could put a link here to an issue or conversation doing that it'd be awesome.
> 
> Similarly, we need to send the appdata file upstream, it's generally useful for all Linux distribution types, not just flatpaks and we don't want to host 'improvements' or changes here where we can help it.

Yes im go to ask from the dev